### DISCUSSION TITLE
`brew doctor` complains of unbrewed header files in `/usr/local` despite being installed in `/opt/homebrew` prefix

### DISCUSSION THREAD

**Discussion by alexreg** [Homebrew/discussions#3101] [Everyday usage]
Running `brew doctor` gives me output that includes the following, complaining of "unbrewed header files" in `/usr/local`, even though I'm running on arm64, and everything brew-related is in `/opt/homebrew`. Surely this isn't right?

```
Warning: Unbrewed header files were found in /usr/local/include.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected header files:
  /usr/local/include/BandedSmithWaterman.h
  /usr/local/include/BedReader.h
  /usr/local/include/Fasta.h
  ...
```

**Comment by SMillerDev** [comment]
`/usr/local` is still a standard location for files, and it'll still be picked up by default. That's actually why Homebrew initially picked it to install in, but that also means that it can be a problem when files are there regardless of where Homebrew is.

**Reply by alexreg** [reply]
Hmm, okay. That's slightly annoying/unexpected still, since lots of other manually-installed programs put stuff in `/usr/local`. I thought that finally with Homebrew moving to `/opt/homebrew` we could get a clean separation of the two...
